backend/spec/system/front_end/survey_crud_spec.rb

```diff
@@ -25,6 +25,37 @@
 
         it { expect(page).to have_text('2 questions answered') }
       end
+
+      context 'with single questions' do
```

Comment:
Please add a case where a survey has one single_choice question and one multiple_choice question.